frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -215,6 +220,24 @@ const ComparisonSeries = ({
     setDialogOpen(false);
   };
 
+  const skipTutorial = () => {
+    closeDialog();
+
+    setTutorialIsSkipped(true);
+    window.localStorage.setItem('tutorialIsSkipped', 'true');
```

Comment:
Let's track this event in Plausible.

We can add the following line at the end of the `skipTutorial` funcction (we also need to update `TRACKED_EVENTS`).

```TypeScript
trackEvent(TRACKED_EVENTS.tutorialSkipped, { props: { step: step } });
```

```TypeScript
export const TRACKED_EVENTS = {
  signup: 'signup',
  tutorial: 'tutorial',
  tutorialSkipped: 'tutorial skipped',
  accountDeleted: 'account deleted',
};
```

Reply:
fixed by > bd1c895